### DISCUSSION TITLE
Will this project become fully open-source?

### DISCUSSION THREAD

**Discussion by bren96** [EsriPS/arcgis-assistant-feedback#19] [Q&A] [answered]
Will this project become fully open-source? The previous [ago-assistant](https://github.com/Esri/ago-assistant) is a great project, really glad that it is open-source. Will this revamp also become open-source? Regardless, thank you for making some fantastic tools!

**Comment by jpeterson** [comment, marked as answer]
@bren96 yes! 

The plan is to fully open-source this version of Assistant as well. We are working through a few things in order to make that happen (primarily ironing out an [on-prem/disconnected deployment strategy](https://guide.assistant.esri-ps.com/docs/faq#can-i-install-arcgis-assistant-on-my-own-server)).

**Reply by widsuoids** [reply]
What is the progress now, one year later?

**Reply by jpeterson** [reply]
@widsuoids thanks for checking in on this. We're still making progress toward open-sourcing ArcGIS Assistant. Some team shuffling and other priorities have slowed us down a bit over the past year. The team unfortunately can't commit to a specific date, but this is still very much the plan.

**Reply by brian32768** [reply]
Just call it version 0.1. It does not matter if it is highly polished. Think of all the advantages. You won't have to keep making excuses for more years, and you can just release it today. Let one of the interns do it. Then we can just fix bugs and make improvements ourselves. The code will get better! No charge to you!! Please tell your management that you figured out a clever way to get people to work for you for free.

The irony here is that you called the version you released "Assistant BETA". The other irony is we already assume every Esri product is "BETA" so actually calling it that is meaningless. ;-)

**Reply by widsuoids** [reply]
@jpeterson ...3 years have now passed since the question was first asked...
